src/parser/shaman/elemental/modules/checklist/Component.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SPELLS from 'common/SPELLS';
+import Checklist from 'parser/shared/modules/features/Checklist2';
+import Rule from 'parser/shared/modules/features/Checklist2/Rule';
+import PreparationRule from 'parser/shared/modules/features/Checklist2/PreparationRule';
+import GenericCastEfficiencyRequirement from 'parser/shared/modules/features/Checklist2/GenericCastEfficiencyRequirement';
+
+class ElementalShamanChecklist extends React.PureComponent {
+  static propTypes = {
+    castEfficiency: PropTypes.object.isRequired,
+    combatant: PropTypes.shape({
+      hasTalent: PropTypes.func.isRequired,
+      hasTrinket: PropTypes.func.isRequired,
+    }).isRequired,
+    thresholds: PropTypes.object.isRequired,
+  };
+
+  render() {
+    const { combatant, castEfficiency, thresholds } = this.props;
+    const AbilityRequirement = props => (
+      <GenericCastEfficiencyRequirement
+        castEfficiency={castEfficiency.getCastEfficiencyForSpellId(props.spell)}
+        {...props}
+      />
+    );
+    return (
+      <Checklist>
+        <Rule
+          name="Use core abilities as often as possible"
+          description={(
+            <>
+              Using your core abilities as often as possible can help raise your dps significantly.Some help more than others, but as a general rule of thumb you should be looking to use most of your damaging abilities and damage cooldowns as often as possible, unless you need to save them for a priority burst phase that is coming up soon.
+              {'  '}
```

Comment:
Have you tried using unbreakable spaces (`&nbsp;`) instead of this? Feels cleaner to me that way.

Reply:
I replaced it with a "proper" non-breaking space but using &nbsp; in JSX I can't get to work.